### ISSUE TITLE
Need a formula for rebar3

### ISSUE THREAD

**Issue by ijt** [Homebrew/homebrew#38306]
Rebar3 is the new version of the main build tool for Erlang. http://www.rebar3.org/v3.0/docs/faq

It would be nice to have a formula for it, but brew doesn't seem to be able to recognize the difference between rebar and rebar3.

$ brew create https://github.com/rebar/rebar3.git
Error: /usr/local/Library/Formula/rebar.rb already exists

**Comment by tdsmith** [comment]
Try pointing it at a release tarball.

If this will supplant rebar, we can add a `devel` block to the rebar formula.

If this needs to exist in parallel to rebar, this can go in `homebrew-head-only` or `-devel-only` for now, or else we can wait for a stable release before adding it to core.

**Comment by ijt** [comment]
There aren't any release tarballs yet, as far as I can tell, though there
are nightly builds. I'm adding a head block.

**Comment by tdsmith** [comment]
They're linked here: https://github.com/rebar/rebar3/releases

**Comment by DomT4** [comment]
Ideally, would be added as a ` devel ` block. ` devel-only ` is intended primarily for things that have absolutely no stable release yet, which is why it's the easiest and most low-volume repo in the world to maintain :smile_cat:.

**Comment by DomT4** [comment]
Closing per Tim's earlier comment. PR welcome in line with that.
